AUTHORS.md

```diff
@@ -5,13 +5,13 @@ during [an NSF funded postdoc](http://www.leouieda.com/blog/hawaii-gmt-postdoc.h
 with [Paul Wessel](http://www.soest.hawaii.edu/wessel) at the University of Hawaii at
 Manoa.
 
-The following people have contributed code to the project (alphabetical by last name)
-and are considered the "PyGMT Developers":
+The following people have contributed code to the project (alphabetical by name)
+and are considered to be "PyGMT Developers":
 
-* [Michael Grund](https://github.com/michaelgrund)
-* [Wei Ji Leong](https://github.com/weiji14)
-* [Tyler Newton](http://www.tnewton.com/)
-* [William Schlitzer](https://github.com/willschlitzer)
-* [Dongdong Tian](https://seisman.info/)
-* [Liam Toney](https://liam.earth/)
+* [Dongdong Tian](https://seisman.info/) | [0000-0001-7967-1197](https://orcid.org/0000-0001-7967-1197) | Department of Earth and Environmental Sciences, Michigan State University, East Lansing, MI, USA
 * [Leonardo Uieda](http://www.leouieda.com/)
+* [Liam Toney](https://liam.earth/) | [0000-0003-0167-9433](https://orcid.org/0000-0003-0167-9433) | Alaska Volcano Observatory and Wilson Alaska Technical Center, Geophysical Institute, University of Alaska Fairbanks, Fairbanks, AK, USA
+* [Michael Grund](https://github.com/michaelgrund) | [0000-0001-8759-2018](https://orcid.org/0000-0001-8759-2018)
```

Comment:
> Just want to notify @carocamargo, @mjziebarth, and @akshmakov of this PR, in case they want to take the opportunity to add themselves to the author list for their wrapping of grdfilter, contour, and grdcontour respectively.

Thanks @meghanrjones for the ping! Looking at the authorship policy, I'm glad to supply my information (even though it's been only one small contribution):

```suggestion
* [Malte Ziebarth](https://github.com/mjziebarth) | [0000-0002-5190-4478](https://orcid.org/0000-0002-5190-4478) | GFZ German Research Centre for Geosciences, Potsdam, Germany
* [Michael Grund](https://github.com/michaelgrund) | [0000-0001-8759-2018](https://orcid.org/0000-0001-8759-2018)
```

I have not been able to follow the instructions that @tjnewton citet due to access rights (not sure if mine differ), so I followed @weiji14's alternative suggestion.